### PR TITLE
Accessibility enhancement for code editor icons

### PR DESCRIPTION
**Pull request by ayumihamsaki**
This PR fixes the issue of the code editor icons being hard to see, they now match the other icons and this PR only works in the `CMS` web page section and not the rest of the cms which when testing is ok.

See test results on a live test server:

Untitled

@w20k ready for you to test and check, shouldn't take up too much of your time, as I've double checked this for you.

Fixes Github issue: https://github.com/octobercms/october/issues/4392

### CODE REVIEW

File: modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less [hunk 1]
```diff
@@ -119,4 +120,18 @@
...
+			>ul {
+				>li {
+					>a {
+						color: @color_3;
+					}
+				}
+			}
+		}
+	}
 }
```
Comment (LukeTowers) [inline modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less]: It's not necessary to have this much nesting for only one rule. I would prefer that it was all one line for the selector this case as it's only used for one rule.

Comment (ayumihamsaki) [reply modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less]: Please could you check my update and let me know if that's ok or not? I was not sure about spacing the brackets.

File: modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less [hunk 2]
```diff
@@ -119,4 +120,6 @@
         }
     }
 
-}
\ No newline at end of file
+}
+
+#cms-master-tabs { .field-codeeditor { .editor-toolbar { >ul { >li { >a { color: @color_3; } } } } } }
```
Comment (w20k) [inline modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less]: @ayumihamsaki I think @LukeTowers meant something like that:
```suggestion
#cms-master-tabs .field-codeeditor .editor-toolbar > ul  > li  > a { 
    color: @color_3; 
} 
```

Comment (ayumihamsaki) [reply modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less]: I have to laugh! What was I thinking!

**Comment by w20k** [comment]
Done - ✅ 

Screenshot:
Screenshot 2019-07-08 at 10 49 09

**Pull request merged**